package.json

```diff
@@ -18,7 +18,7 @@
     "Richard Chamberlain <richard_chamberlain@uk.ibm.com> (https://github.com/rnchamberlain)"
   ],
   "scripts": {
-    "test": "tap test/test*.js"
+    "test": "make lint && tap test/test*.js"
```

Comment:
I agree if we can use a Node.js script instead of make that would simply supporting it across platforms.